contracts/core/extensions/CoreAccounting.sol

```diff
@@ -118,13 +118,22 @@ contract CoreAccounting is
         nonReentrant
     {
         // Confirm an empty _tokens array is not passed
-        require(_tokens.length > 0, "WITHDRAW_TOKENS_ARRAY_EMPTY");
+        require(
+            _tokens.length > 0,
+            "Core.batchWithdraw: Empty tokens array"
```

Comment:
Change this to "Tokens must not be empty"? Same with following error. I'm a fan of errors saying "X must be Y" instead of "X is Z".